src/ucp/wireup/wireup_cm.c

```diff
@@ -158,9 +158,36 @@ ucp_cm_tl_bitmap_get_dev_idx(ucp_context_h context, ucp_tl_bitmap_t tl_bitmap)
     return dev_index;
 }
 
+static void
+ucp_cm_tl_bitmap_single_tl(ucp_context_h context, ucp_tl_bitmap_t *tl_bitmap)
```

Comment:
maybe `*first_tl`? BTW, can we select "best" instead of "first"? then it could be single...

Reply:
done